Package/ExcelDna.AddIn/tools/ExcelDna.targets

```diff
@@ -0,0 +1,291 @@
+
+  <!--
+    Set default values for configuration properties that have not been set
+  -->
+  <PropertyGroup>
+    <RunExcelDnaBuild Condition="'$(RunExcelDnaBuild)' == ''">true</RunExcelDnaBuild>
+    <RunExcelDnaPack Condition="'$(RunExcelDnaPack)' == ''">true</RunExcelDnaPack>
+
+    <ExcelDnaToolsPath Condition="'$(ExcelDnaToolsPath)' == ''">$(MSBuildThisFileDirectory)</ExcelDnaToolsPath>
+    <ExcelDnaPackExePath Condition="'$(ExcelDnaPackExePath)' == ''">$(ExcelDnaToolsPath)ExcelDnaPack.exe</ExcelDnaPackExePath>
+
+    <ExcelDnaBuild32Dna Condition="'$(ExcelDnaBuild32Dna)' == ''">true</ExcelDnaBuild32Dna>
+    <ExcelDnaBuild64Dna Condition="'$(ExcelDnaBuild64Dna)' == ''">true</ExcelDnaBuild64Dna>
+
+    <ExcelDnaCreate64DnaFrom32Dna Condition="'$(ExcelDnaCreate64DnaFrom32Dna)' == ''">true</ExcelDnaCreate64DnaFrom32Dna>
+    <ExcelDnaCreate32DnaFrom64Dna Condition="'$(ExcelDnaCreate32DnaFrom64Dna)' == ''">true</ExcelDnaCreate32DnaFrom64Dna>
+
+    <ExcelDna32PlatformId Condition="'$(ExcelDna32PlatformId)' == ''">x86</ExcelDna32PlatformId>
+    <ExcelDna64PlatformId Condition="'$(ExcelDna64PlatformId)' == ''">x64</ExcelDna64PlatformId>
+
+    <ExcelDna32FileSuffix Condition="'$(ExcelDna32FileSuffix)' == ''"></ExcelDna32FileSuffix>
+    <ExcelDna64FileSuffix Condition="'$(ExcelDna64FileSuffix)' == ''">64</ExcelDna64FileSuffix>
+
+    <ExcelDnaPackFileSuffix Condition="'$(ExcelDnaPackFileSuffix)' == ''">-packed</ExcelDnaPackFileSuffix>
+
+    <ExcelDnaXll32File Condition="'$(ExcelDnaXll32File)' == ''">ExcelDna.xll</ExcelDnaXll32File>
+    <ExcelDnaXll32Path Condition="'$(ExcelDnaXll32Path)' == ''">$(ExcelDnaToolsPath)$(ExcelDnaXll32File)</ExcelDnaXll32Path>
+
+    <ExcelDnaXll64File Condition="'$(ExcelDnaXll64File)' == ''">ExcelDna64.xll</ExcelDnaXll64File>
+    <ExcelDnaXll64Path Condition="'$(ExcelDnaXll64Path)' == ''">$(ExcelDnaToolsPath)$(ExcelDnaXll64File)</ExcelDnaXll64Path>
+  </PropertyGroup>
+
+  <!--
+    Target that removes all .dna, .xll, and .xll.config from the build output folder
+  -->
+  <Target Name="ExcelDnaClean">
+    <ItemGroup>
+      <ExcelDnaFilesToDelete Include="$(TargetDir)**\*.dna" />
+      <ExcelDnaFilesToDelete Include="$(TargetDir)**\*.xll" />
+      <ExcelDnaFilesToDelete Include="$(TargetDir)**\*.xll.config" />
+    </ItemGroup>
+
+    <Delete Files="@(ExcelDnaFilesToDelete)"/>
+  </Target>
+
+  <!--
+    Target that gets builds a list of .dna files to be processed by platform
+  -->
+  <Target Name="ExcelDnaPrepare"
+          Condition="$(RunExcelDnaBuild)">
+    <!--
+      Make sure the ExcelDnaPack utility exists
+    -->
+    <Error Text="ExcelDnaPack.exe not found: $(ExcelDnaPackExePath)"
+           Condition="!Exists('$(ExcelDnaPackExePath)')" />
+
+    <!--
+      Make sure the suffix for 32 and 64-bit add-ins are different, so we don't get clashes
+    -->
+    <Error Text="ExcelDna32FileSuffix and ExcelDna32FileSuffix cannot be identical. Fix you ExcelDna.props file"
```

Comment:
Error text should be ExcelDna32FileSuffix and **ExcelDna64FileSuffix** cannot be identical. Fix **your** ExcelDna.props file